build.gradle

```diff
@@ -47,7 +47,7 @@ dependencies {
   jenkinsPlugins 'org.jenkins-ci.plugins:credentials:1.22@jar'
 
   optionalJenkinsPlugins 'org.jenkins-ci.plugins:junit:1.6@jar'
-  optionalJenkinsPlugins 'org.jenkins-ci.plugins:cobertura:1.9.6@jar'
+  optionalJenkinsPlugins 'org.jenkins-ci.plugins:cobertura:1.11@jar'
```

Comment:
Why is this required? Can we decouple this change?

Reply:
Answered in https://github.com/uber/phabricator-jenkins-plugin/pull/185#issuecomment-389227543

Reply:
sounds good